### PR TITLE
 JBPM-7661: NodeInstanceLog for node left events should be fired on b…

### PR DESCRIPTION
**Pull request by nmirasch**
…eforeNodeLeft
depends on https://github.com/kiegroup/jbpm/pull/1318

### CODE REVIEW

**Comment by nmirasch** [comment]
Fixing integration test related with the sorting of the complete nodes retrieved by findCompletedNodeInstances service

**Comment by nmirasch** [comment]
@MarianMacik could you take a look?

File: kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/RuntimeDataServiceIntegrationTest.java [hunk 1]
```diff
@@ -1181,12 +1181,12 @@ public void testGetNodeInstances() throws Exception {
...
 
-            nodeInstance = instances.get(0);
+            nodeInstance = instances.get(1);
             assertNodeInstance(expectedStart, nodeInstance);
             assertNull(nodeInstance.getWorkItemId());
             assertNotNull(nodeInstance.getDate());
 
-            nodeInstance = instances.get(1);
+            nodeInstance = instances.get(0);
             assertNodeInstance(expectedFirstTask, nodeInstance);
```
[the remark this answers is not in the document]
Comment (nmirasch) [reply kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/RuntimeDataServiceIntegrationTest.java]: makes sense!

**Comment by nmirasch** [comment]
@MarianMacik changed, Thanks!

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

**Comment by MarianMacik** [comment]
jenkins retest this

**Pull request merged**